src/core/ddsc/tests/test_oneliner.h

```diff
@@ -303,6 +309,7 @@ struct oneliner_ctx {
   ddsrt_cond_t g_cond;
   struct oneliner_cb cb[3];
   
+  bool disable_sharedmemory;
```

Comment:
This variable seems to be unused (or am I overlooking something?)

Reply:
That is indeed a remnant of an experiment. I'll delete it and force push the updated version. Thanks!